models/transfers/celo/erc20/transfers_celo_erc20_agg_day.sql

```diff
@@ -0,0 +1,32 @@
+{{ 
+    config(
+        tags = ['dunesql'],
+        alias = alias('erc20_agg_day'),
+        partition_by = ['block_month'],
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['wallet_address', 'token_address', 'block_day'],
+        post_hook='{{ expose_spells(\'["celo"]\',
+                                    "sector",
+                                    "transfers",
+                                    \'["tomfutago"]\') }}'
+    )
+}}
+
+select
+  'celo' as blockchain,
+  cast(date_trunc('month', tr.block_time) as date) as block_month,
+  date_trunc('day', tr.block_time) as block_day,
+  tr.wallet_address,
+  tr.token_address,
+  cast(null as varchar(10)) as symbol, -- t.symbol, -- null until tokens_celo_erc20 spell is created
+  sum(tr.amount_raw) as amount_raw,
+  sum(tr.amount_raw / power(10, 18)) as amount --sum(tr.amount_raw / power(10, t.decimals)) as amount
+from {{ ref('transfers_celo_erc20') }} tr
+--left join ref('tokens_celo_erc20') t on t.contract_address = tr.token_address
```

Comment:
this is now updated, do you want to finalize this PR?

Reply:
sure, will do